test/e2e/opentelemetry/instrumentation/opentelemetry.test.ts

```diff
@@ -172,8 +172,8 @@ describe('opentelemetry', () => {
                           'next.clientComponentLoadCount': isNextDev
                             ? // In dev, additional client components are being loaded
                               // due to RSC props being deserialized.
-                              9
-                            : 6,
+                              11
```

Comment:
also curious why this would have changed

Reply:
2 new client components StreamingMetadata and StreamingMetadataOutlet